website/static/js/pages/profile-settings-page.js

```diff
@@ -6,3 +6,18 @@ new profile.Names('#names', ctx.nameUrls, ['edit']);
 new profile.Social('#social', ctx.socialUrls, ['edit']);
 new profile.Jobs('#jobs', ctx.jobsUrls, ['edit']);
 new profile.Schools('#schools', ctx.schoolsUrls, ['edit']);
+
+// Reusable function to fix affix widths to columns.
+function fixAffixWidth() {
+    $('.affix, .affix-top, .affix-bottom').each(function (){
```

Comment:
Would you mind explaining what this function is for? Nothing seems to change when I take it out.